refinery/tool_manager/tests.py

```diff
@@ -2865,9 +2866,7 @@ def test_relaunch_failure_tool_doesnt_exist(self):
         get_request = self.factory.get(relaunch_url)
         force_authenticate(get_request, self.user)
         get_response = self.tool_relaunch_view(get_request, uuid=tool_uuid)
-        self.assertEqual(get_response.status_code, 400)
-        self.assertIn("Couldn't retrieve VisualizationTool",
-                      get_response.content)
+        self.assertEqual(get_response.status_code, 404)
```

Comment:
I think the underlying error message should still be coming through... but if a 404 doesn't seem appropriate, let me know?

Reply:
Now that we use `django.shortcuts.get_object_or_404` a resonable error message is returned. Something like `No VisualizationTool matches the given query.` I was hesitant to test Django internals